src/SPC/builder/unix/library/libtiff.php

```diff
@@ -16,24 +18,31 @@ trait libtiff
      */
     protected function build(): void
     {
+        $libcpp = SPCTarget::getTargetOS() === 'Linux' ? '-lstdc++' : '-lc++';
+        FileSystem::replaceFileStr($this->source_dir . '/configure', '-lwebp', '-lwebp -lsharpyuv');
+        FileSystem::replaceFileStr($this->source_dir . '/configure', '-l"$lerc_lib_name"', '-l"$lerc_lib_name" ' . $libcpp);
```

Comment:
Is this modification necessary? I tested it on my mac and it seemed to work the same with or without it.

Reply:
yes, it fails linking otherwise. lwebp requires lsharpyuv, but the config.m4 doesn't properly call pkgconfig and only adds -lwebp.

lerc also requires c++, but the makefile uses CC to build instead of CXX, leading to undefined references to c++ symbols